src/Form/Type/UiElement/CaseStudiesUiElementType.php

```diff
@@ -23,15 +23,15 @@
 #[AsUiElement(
     code: 'monsieurbiz_blog.case_studies_ui_element',
     icon: 'crosshairs',
-    uiElement: 'MonsieurBiz\SyliusBlogPlugin\UiElement\CaseStudiesUiElement',
     title: 'monsieurbiz_blog.ui_element.case_studies_ui_element.title',
     description: 'monsieurbiz_blog.ui_element.case_studies_ui_element.description',
+    uiElement: 'MonsieurBiz\SyliusBlogPlugin\UiElement\CaseStudiesUiElement',
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Why the lines moved ?